package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@percy/protractor",
+  "version": "0.0.1",
+  "description": "Percy.io integration with Protractor",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/percy/percy-protractor.git"
+  },
+  "author": "Perceptual Inc.",
+  "license": "MIT",
+  "devDependencies": {
+    "@percy/agent": "^0.1.3",
+    "typescript": "^3.1.6"
+  },
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "build": "tsc"
+  },
+  "peerDependencies": {},
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@percy/tslint": "^1.0.0"
```

Comment:
I think this should be under `devDependencies`

Reply:
Ah, yes you're right.

Reply:
Fixed. And also made `@percy/agent` a prod dependency, out of `devDependencies`.